swagger_server/controllers/operator_controller.py

```diff
@@ -22,7 +34,11 @@ def delete_operator(operator_id):  # noqa: E501
 
     :rtype: None
     """
-    return 'do some magic!'
+    try:
+        operand = Operator.delete().where(Operator.id == operator_id)
+    except Exception as exc:
+        raise ValidationError(400, 'Expression not found!')
+    return flask.jsonify({'expression_id': operator_id})
```

Comment:
I think you can just return a jsonified True here if the delete succeeded and a 404 validation error if it wasn't able to delete stuff.